src/ext/java/org/opentripplanner/ext/ridehailing/service/oauth/OAuthService.java

```diff
@@ -0,0 +1,7 @@
+package org.opentripplanner.ext.ridehailing.service.oauth;
+
+import java.io.IOException;
+
+public interface OAuthService {
+  String getToken() throws IOException;
```

Comment:
All public classes and method should have Java documentation.

Reply:
Done in 84ceac77ac8788fa68bdc8b5d2951ae6a72edca0